core/tests/protractor_desktop/learnerDashboard.js

```diff
@@ -136,6 +136,47 @@ describe('Learner dashboard functionality', function() {
     users.logout();
   });
 
+  it('should add exploration to play later list', function() {
+    var EXPLORATION_FRACTION = 'fraction';
+    var EXPLORATION_SINGING = 'singing';
+    var CATEGORY_MATHEMATICS = 'Mathematics';
+    var CATEGORY_MUSIC = 'Music';
+    var LANGUAGE_ENGLISH = 'English';
+    var EXPLORATION_OBJECTIVE = 'hold the light of two trees';
+    var EXPLORATION_OBJECTIVE2 = 'show us the darkness';
+
+    users.createUser(
+      'creator@learnerDashboard.com', 'creatorLearnerDashboard');
+    users.login('creator@learnerDashboard.com');
+    workflow.createAndPublishExploration(
+      EXPLORATION_FRACTION, CATEGORY_MATHEMATICS,
+      EXPLORATION_OBJECTIVE, LANGUAGE_ENGLISH);
+    workflow.createAndPublishExploration(
+      EXPLORATION_SINGING, CATEGORY_MUSIC,
+      EXPLORATION_OBJECTIVE2, LANGUAGE_ENGLISH);
+    users.logout();
+    users.createUser(
+      'learner@learnerDashboard.com', 'learnerLearnerDashboard');
+    users.login('learner@learnerDashboard.com');
+    libraryPage.get();
+    libraryPage.findExploration('fraction');
+
+    libraryPage.addToPlaylist(0);
+    learnerDashboardPage.get();
+    learnerDashboardPage.navigateToPlayLaterExplorationSection();
+    learnerDashboardPage.expectTitleOfExplorationSummaryTileToMatch(
+      EXPLORATION_FRACTION, 0);
+
+    libraryPage.get();
+    libraryPage.findExploration('singing');
+    libraryPage.addToPlaylist(0);
```

Comment:
Why we are passing `int` to the `addToPlaylist` method?

Reply:
Now we are not, see the lower comment. So, Done